app/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -799,7 +806,7 @@ public void onScaleChanged( float scale, float previous ) {
 
   @Override
   public void onPanBegin( int x, int y, Origination origin ) {
-    suppressRender();
```

Comment:
This removal is due to the `requestRender` change, which no longer sets `mRenderIsSuppressed` to false.
Here i am questioning the necessity of having `suppressRender` at the beginning of a pan.
If it turns out that this was necessary, we can still add it back and add a `resumeRender` in `onPanUpdate` method.